server/backend/api/app/routers/router_guards.py

```diff
@@ -136,10 +136,13 @@ def authorize_by_app_id(
     db: Session = Depends(database.get_db),
     access_roles=Depends(get_access_roles),
     requester: RequesterSchema = Depends(get_current_requester),
-):
+) -> int:
     """
     This authorize_by_app_id method is used for the authorization check of a specific application,
     we require user to be the app admin or delegated admin of the application
+
+    Returns:
+        int: The application ID if the user is authorized.
```

Comment:
Hi Craig, since there is not need for `application_id` return, the statement can be removed and the also the return type `-> int`